tests/src/JIT/HardwareIntrinsics/X86/Sse2/AddSaturate.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace IntelHardwareIntrinsicTest
+{
+    internal static partial class Program
```

Comment:
why internal?

Reply:
It's not used outside of the assembly, but I am entirely indifferent to how accessibility is defined